qadence/ml_tools/printing.py

```diff
@@ -1,7 +1,23 @@
 from __future__ import annotations
 
+from logging import getLogger
+from typing import Any, Callable, Union
+
+from matplotlib.figure import Figure
+from mlflow.models import infer_signature
+from torch import Tensor
+from torch.nn import Module
+from torch.utils.data import DataLoader
 from torch.utils.tensorboard import SummaryWriter
 
+from qadence.ml_tools.data import DictDataLoader
+from qadence.types import ExperimentTrackingTool
+
+logger = getLogger(__name__)
+
+PlottingFunction = Callable[[Module, int], tuple[str, Figure]]
+InputData = Union[Tensor, dict[str, Tensor]]
```

Comment:
```suggestion
InputData = Tensor | dict[str, Tensor]
```

Reply:
Had to actually revert this in fdf05f1, because of invalid type definition error raised by mypy.